src/wdio/services/AxeService.js

```diff
@@ -33,7 +32,7 @@ export default class AxeService {
       const specifiedViewports = options.viewports || [currentViewportSize];
       const axeOptions = {
         runOnly: options.runOnly,
-        rules: options.rules,
+        rules: Object.assign({}, axeConfig.options.rules, options.rules),
```

Comment:
the axeConfig should be set globally using `axe.configure` when the script is injected.